server/game/gameSystems/DefeatCardSystem.ts

```diff
@@ -35,12 +36,11 @@ export class DefeatCardSystem<TContext extends AbilityContext = AbilityContext,
 
         if (event.card.isToken()) {
             // move the token out of the play area so that effect cleanup happens, then remove it from all card lists
-            event.card.owner.moveCard(event.card, ZoneName.OutsideTheGame, event.options || {});
-            event.context.game.removeTokenFromPlay(event.card);
+            event.card.moveTo(ZoneName.OutsideTheGame);
```

Comment:
Would it make any sense to put a .defeat() method on the card that moves itself based on type?